src/objects/BDI_Field_Mapping__mdt.object

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Batch Data Import takes a single NPSP Data Import record and translates it into multiple records (Contacts, Accounts, Opportunity, Payment, etc.). It does this by mapping fields from the Source Field to the Target Field.</description>
+    <fields>
+        <fullName>Source_Field__c</fullName>
+        <caseSensitive>false</caseSensitive>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <label>Source Field</label>
+        <length>255</length>
+        <required>true</required>
+        <type>Text</type>
+        <unique>true</unique>
```

Comment:
We'll have to change this to false to accommodate mapping to multiple targets.

Reply:
Not resolved yet, but this sounds like a future-looking comment?